src/lightning/pytorch/utilities/model_summary/model_summary.py

```diff
@@ -392,8 +396,8 @@ def get_human_readable_count(number: int) -> str:
     """
     assert number >= 0
     labels = PARAMETER_NUM_UNITS
-    num_digits = int(np.floor(np.log10(number)) + 1 if number > 0 else 1)
-    num_groups = int(np.ceil(num_digits / 3))
+    num_digits = int(torch.floor(torch.log10(torch.tensor(number, dtype=torch.float32))) + 1 if number > 0 else 1)
+    num_groups = int(torch.ceil(torch.tensor(num_digits, dtype=torch.float32) / 3))
```

Comment:
We don't need to convert to tensors here. Let's simplify it like this?
```suggestion
    num_digits = int(math.floor(math.log10(number)) + 1 if number > 0 else 1)
    num_groups = int(math.ceil(num_digits / 3))
```